packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-modal-content.spec.tsx

```diff
@@ -0,0 +1,315 @@
+                    },
+                    signup: [''],
+                },
+                shortcode: Jurisdiction.VANUATU,
+                sub_account_type: '',
+            },
+        ],
+        financial_available_accounts: [
+            {
+                market_type: 'financial' as const,
+                name: '',
+                requirements: {
+                    after_first_deposit: {
+                        financial_assessment: [''],
+                    },
+                    compliance: {
+                        mt5: [''],
+                        tax_information: [''],
+                    },
+                    signup: [''],
+                },
+                shortcode: Jurisdiction.SVG,
+                sub_account_type: '',
+            },
+            {
+                market_type: 'financial' as const,
+                name: '',
+                requirements: {
+                    after_first_deposit: {
+                        financial_assessment: [''],
+                    },
+                    compliance: {
+                        mt5: [''],
+                        tax_information: [''],
+                    },
+                    signup: [''],
+                },
+                shortcode: Jurisdiction.BVI,
+                sub_account_type: '',
+            },
+            {
+                market_type: 'financial' as const,
+                name: '',
+                requirements: {
+                    after_first_deposit: {
+                        financial_assessment: [''],
+                    },
+                    compliance: {
+                        mt5: [''],
+                        tax_information: [''],
+                    },
+                    signup: [''],
+                },
+                shortcode: Jurisdiction.VANUATU,
+                sub_account_type: '',
+            },
+            {
+                market_type: 'financial' as const,
+                name: '',
+                requirements: {
+                    after_first_deposit: {
+                        financial_assessment: [''],
+                    },
+                    compliance: {
+                        mt5: [''],
+                        tax_information: [''],
+                    },
+                    signup: [''],
+                },
+                shortcode: Jurisdiction.LABUAN,
+                sub_account_type: '',
+            },
+        ],
+        context: mock_context,
+        real_synthetic_accounts_existing_data: [],
+        real_financial_accounts_existing_data: [],
+        real_swapfree_accounts_existing_data: [],
+        swapfree_available_accounts: [],
+        card_flip_status: {
+            svg: false,
+            bvi: false,
+            vanuatu: false,
+            labuan: false,
+            maltainvest: false,
+        },
+        flipCard: jest.fn(),
+    };
+
+    it('should display cfd-jurisdiction-card--synthetic__wrapper in class name', () => {
```

Comment:
Are we covering the test case for Swap-free in this file?

Reply:
@hamza-deriv added test cases for swap-free too.